pages/dashboards/templates/dashboards/multidisease_serology.html

```diff
@@ -0,0 +1,164 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <a href="#content-start" class="sr-only focus:not-sr-only focus:outline-none focus:ring-2 focus:ring-pp-mid-blue px-3 py-2 bg-white">Skip to content</a>
+  <h1 class="sr-only">Multi-disease serology</h1>
+  <div id="content-start">
+    <div class="prose max-w-none space-y-6 mt-4">
+      {% comment %}
+      <div class="bg-pp-pale-grey/30 border-l-4 border-pp-mid-blue text-pp-dark-blue p-4" role="status" aria-live="polite" aria-atomic="true">
+        <p class="m-0"><strong>All data last updated:</strong> TODO</p>
+      </div>
+      {% endcomment %}
+
+      <h2 id="introduction">Introduction</h2>
+      <p>
+        The COVID-19 pandemic highlighted the importance of serological surveillance in tracking viral
+        transmission dynamics, understanding immune responses, guiding vaccination strategies, and assisting
+        in decisions related to public health. High‑throughput serological assays for SARS‑CoV‑2 were
+        developed very early in the pandemic at KTH and SciLifeLab to enable surveillance of populations
+        globally. For information about work done with SARS‑CoV‑2 during the pandemic, see the
+        <a href="/historical-background/">historical background section</a>.
+      </p>
+      <p>
+        As we move on from the pandemic, it is crucial that serological studies include more pathogens
+        than just SARS‑CoV‑2. One of the capabilities that is part of
+        <a href="/plp-program-background/">SciLifeLab's Pandemic Laboratory Preparedness (PLP) programme</a>,
+        named 'Multi‑disease serology' aims to create a sustainable, long‑term
+        resource enabling a broad, frequent, and large‑scale surveillance of serostatus. They are working to
+        create antibody repertoires that can be used to analyse thousands of samples on hundreds of antigens.
+        The project is led by Peter Nilsson at KTH Royal Institute of Technology and SciLifeLab. To learn more,
+        check out the <a href="/resources/serology/">pandemic preparedness resource page for this project</a>.
+      </p>
+
+      <h2 id="historical-background">Historical background</h2>
+      <p>
+        During the pandemic, those working on the multi‑disease serology study created a high‑throughput
+        multiplex bead‑based serological assay for SARS‑CoV‑2 (see
+        <a href="https://doi.org/10.1002/cti2.1312" target="_blank" rel="noopener noreferrer">Hober <em>et&nbsp;al.</em> (2021) <span class="sr-only">(opens in a new tab)</span></a>
+        and the <a href="/dashboards/serology-statistics/">Serology tests for SARS‑CoV‑2 at SciLifeLab Data Dashboard</a>
```

Comment:
this dashboard is merged and exists, so you can already use `{% url "dashboards:serology_statistics" %}`. 

Or change all links together later, just let you know :)